test/nodes/test_llm_streaming.py

```
@@ -0,0 +1,616 @@
+
+    Returns a module-like namespace with an ``Answer`` class that
+    mirrors the real Answer's ``setAnswer`` / ``getText`` / ``isJson``
+    interface without needing Pydantic or the client SDK.
+    """
+
+    class FakeAnswer:
+        def __init__(self, expectJson=False):
+            self.expectJson = expectJson
+            self.answer = None
+
+        def setAnswer(self, value):
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `setAnswer`

Add return type annotation: `None`

(ANN202)